fdbserver/MoveKeys.actor.cpp

```diff
@@ -1598,6 +1598,11 @@ void seedShardServers(Arena& arena,
 			}
 		}
 
+		// Populate storageTeamIdKey for private teams.
+		for (const auto& [ss, teams] : serverToTeams) {
+			tr.set(arena, storageTeamIdKey(ss), encodeStorageTeams({ ss }));
```

Comment:
a question for me to understand: does `writeToSeedTeamServersDuringClusterSetupTxn(` write(i.e. forward) this mutation to seed team?

Reply:
It should.